### PR TITLE
Fixes SCPs breaking machinery

### LINKED ISSUE #1145: SCPs overriding machinery code
#### Description of issue

SCP code to destroy machinery just manually edits the machines stat instead of using machinery procs to break it. This causes bugs such as icons not properly updated.

#### Difference between expected and actual behavior

Current: A.stat |= BROKEN
Ideal. A.set_broken()

#### Steps to reproduce

Look at SCP code.

#### Specific information for locating

Use the search and replace function on VSC and search for "|= BROKEN"

#### Length of time in which bug has been known to occur

Git blame gives several different dates, ranging from 10 months to 3 weeks,

### PR DESCRIPTION
**Pull request by aidan-bowman**
## About the Pull Request

Closes #1145.

## Why It's Good For The Game

This is shitcode and it breaks icons as well as bypassing special effects of machinery breaking (as well as machinery fixing).

## Changelog

:cl:
fix: SCPs breaking machinery is slightly improved.
/:cl:

### CODE REVIEW

File: code/modules/recycling/conveyor2.dm [hunk 1]
```diff
@@ -198,18 +198,6 @@ GLOBAL_LIST_INIT(conveyor_switches, list())
...
-	stat |= BROKEN
-	operable = FALSE
-	update_icon()
-	var/obj/machinery/conveyor/C = locate() in get_step(src, forwards)
-	if(C)
-		C.set_operable(TRUE, id, FALSE)
-	C = locate() in get_step(src, backwards)
-	if(C)
-		C.set_operable(FALSE, id, FALSE)
-
```
Comment (Tsurupeta) [inline code/modules/recycling/conveyor2.dm]: you sure its not needed and doesnt need to be moved to set_broken????????

Comment (aidan-bowman) [reply code/modules/recycling/conveyor2.dm]: Wasn't called by anything, but it'd probably be wise to make it default behavior. I'll take a look at it.

**Comment by checkraisefold** [comment]
who made this shitcode so we can laugh at them in discord

File: code/modules/recycling/conveyor2.dm (hunk 1 above)
Comment (aidan-bowman) [reply code/modules/recycling/conveyor2.dm]: done

Comment (Tsurupeta) [reply code/modules/recycling/conveyor2.dm]: done what

Comment (Tsurupeta) [reply code/modules/recycling/conveyor2.dm]: your shit doesn't even compile now

Comment (aidan-bowman) [reply code/modules/recycling/conveyor2.dm]: this is what no testing does to a mf

**Review by Tsurupeta — APPROVED** [review]

**Pull request merged**